shared_python/TagValidator.py

```diff
@@ -54,33 +54,26 @@ def identify_category(self, tag):
 
     ## Print statements
     def print_tag_correction(self, before, after):
-        print('\r\033[96m', end='')
         self.log.info('Correction successful. "' + before + '" is now "' + after+ '"')
-        print('\x1b[0m', end='')
 
     def print_tag_warning(self, tag, tag_type, isType):
-        print('\r\033[93m', end='')
+        print()
         if (isType):
```

Comment:
This is something I really should have picked up when I originally reviewed this code, but you shouldn't have brackets around an if condition in Python. (I missed it presumably for the same reason you put them in: too used to reading Java/JavaScript to remember this is Python! 😄 )

Reply:
By brackets, do you mean the parentheses? When I see brackets I think square brackets.

Reply:
Yes indeed, that's what I mean (and there I was using "brackets" as I've had people stare at me blankly when I say "parentheses"!)